manifest.json

```diff
@@ -46,7 +46,7 @@
           }
         }
       },
-      "checkAssignedIssues": {
+      "assignedIssueScope": {
         "default": "org",
         "type": "string"
       },
```

Comment:
are you making this manually? We have a [workflow](https://github.com/ubiquity-os-marketplace/command-start-stop/blob/development/.github/workflows/update-configuration.yml) that automatically updates it, it should include the enum here

Reply:
let the workflow generate it